### ISSUE TITLE
[Issue] Getting the Large Ore Deposits features generating in the mining world

### ISSUE THREAD

**Issue by sciguyryan** [nanite/JAMD#7]
### Minecraft Version

1.19.2

### Modloader

forge-43.2.3

### Mod version

2.0.5-build.12+mc1.19.2

### Repeatable?

- [X] Yes

### Link to log files

Not applicable here.

### Steps to reproduce the issue

1. Attempt to load JAMD with the large ore deposits mod.
2. The do not generate in the mining world.
3. Attempting to add the feature (which as far as I can tell is called adlods:deposit_generator) results in the world being unloadable.

### Description of the issue you are having

Attempting to load JAMD with the large ore deposits mod () doesn't generate the ores in the mining dimension.

I have attempted to solve this using a data pack (as far as I can tell the feature is called adlods:deposit_generator) results in the inability to load the world, something relating to a datapack error.

I've attempted to add this feature under every category in the features JSON file (it's not documented so I have no way of knowing which one it should have been in) results in the same datapack error. I'm somewhat stumped as to how to solve this.

### Information

- [X] I have provided as much information as possible

**Comment by susan3333333** [comment]
Have you tried the lods configs, I don't know if there is a way to allow overall, but each deposit config has a dimension setting like this:


Dimensions
    #--------------------------------------------------------------------------------------------------------#
    # List of dimensions which this ore can be generated in.
    # Dimension is specified by its registry name (e.g: minecraft:overworld, minecraft:the_nether, minecraft:the_end, etc.)
    # Each dimension must be on a separate line without any delimiters.
    # If the whitelist is set, the blacklist will be ignored.
    ##########################################################################################################

    Dimensions {
        S:blackList <
         >
        S:whiteList <
            minecraft:overworld


You just need to find the right name for the mining dimension and add it to the whitelist for the deposits you want to generate.

**Comment by UnRealDinnerbone** [comment]
This can be done with datapack, KubeJS or the config linked above. All I do is provide a dimension in a datapack

**Comment by FoxMcloud5655** [comment]
With version 3.3.0, this no longer works.  The mod now stops all other forms of generation, including from Additional Large Ore Deposits.  This update has broken all generation in my world, and I have to revert to a previous version in order to fix it.

**Comment by FoxMcloud5655** [comment]
I'd like to add that the version listed on CurseForge is 3.4.0, but the changelog states that it's version 3.3.0.

**Comment by UnRealDinnerbone** [comment]
is this the mod you are talking about? https://www.curseforge.com/minecraft/mc-mods/large-ore-deposits

**Comment by FoxMcloud5655** [comment]
Yes, that is the mod I'm currently using for ore generation in all worlds.

**Comment by FoxMcloud5655** [comment]
I find the lack of attention on this issue a little concerning.  A way to toggle off the enforced oregen in the config for JAMD would literally fix this issue instantly.
